app/scenes/channel_info/channel_info.js

```diff
@@ -41,20 +41,44 @@ export default class ChannelInfo extends PureComponent {
         theme: PropTypes.object.isRequired,
         actions: PropTypes.shape({
             getChannelStats: PropTypes.func.isRequired,
-            goToChannelMembers: PropTypes.func.isRequired
+            goToChannelMembers: PropTypes.func.isRequired,
+            markFavorite: PropTypes.func.isRequired,
+            unmarkFavorite: PropTypes.func.isRequired
         })
     }
 
+    constructor(props) {
+        super(props);
+
+        this.state = {
```

Comment:
Yes but the element was jumping back and forth here when it was directly handle by props while redux propagated.  Happy to handle that in another way if you have a suggestion.